.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: Test
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        runs-on: [ubuntu-20.04]
+        tarantool: ['1.10', '2.4', '2.5', '2.6']
```

Comment:
Do you really need all of them for luarapidxml? I think 1.10 and 2.6 is more than enough.

Reply:
BTW, can we run testing on 2.7-live to spot possible problems earlier?

Reply:
Discussed the matrix in chat. The jobs are parallel, all take 50 sec. There's nothing to save.
2.7 added.